src/stores/view.ts

```diff
@@ -1,16 +1,25 @@
 import { create } from 'zustand';
 import { persist, createJSONStorage } from 'zustand/middleware';
+import { View } from '../utils/types';
+
+type NavigationProps = {
+  backView: View | undefined;
+  nextView: View | undefined;
+  backLabel?: string;
+  nextLabel?: string;
+  className: string;
+};
 
 type ViewStore = {
-  currentView: string;
-  setCurrentView: (view: string) => void;
+  currentView: View;
+  setCurrentView: (view: View) => void;
 };
 
 const useViewStore = create<ViewStore>()(
   persist(
```

Comment:
mind removing the `persist` stuff from here as well now? Or is there a reason we're keeping it here?

Reply:
No it's simply because this was branched off main before the other PR that removed it was merged will be removed once we merge main into this branch